### DISCUSSION TITLE
Public directory access in a Vite+React app

### DISCUSSION THREAD

**Discussion by bstro** [vercel/vercel#7804] [Help]
Hi there,

I am using Vercel to host a frontend React app created with Vite, using Vercel's official config for Vite.

I have a static zip file in my public directory that I need to link to from my app. I can access the file from my local dev server (`localhost:3000/public/download.zip`) but not from my vercel deployment. Neither `/download.zip` nor `/public/download.zip` work from my deployment.

I have a `.vercel` configuration that looks like this:
```
{
  "rewrites": [{ "source": "/(.*)", "destination": "/" }]
}
```

Is there something I need to do to this configuration in order to allow my public files to be accessible from my production deployment?

Thanks!

**Comment by correafer** [comment]
any updates? Same problem!

**Reply by bstro** [reply]
I looked for awhile but had to move on to other things. I haven't looked for new information on this in several months, so something could be out there now.

**Comment by amyegan** [comment]
What version of Vite are you using? I found a similar post on Stack Overflow that was solved by upgrading to Vite 3: https://stackoverflow.com/questions/72934786/vite-not-serving-files-from-public-folder-in-production
